### PR TITLE
Improve script

### PR DESCRIPTION
**Pull request by rthomazel**
See comments for explanations. For the `command -v` change see [here](https://stackoverflow.com/questions/592620/how-to-check-if-a-program-exists-from-a-bash-script/677212#677212). I did basic tests.

### CODE REVIEW

File: Run.command [hunk 1]
```diff
@@ -1,27 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (rthomazel) [inline Run.command:1]: more flexible shebang

File: Run.command [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
 
-py="$( which python )"
-py3="$( which python3 )"
+[[ -x "$( command -v python3 )" ]] && python3="true"
```
Comment (rthomazel) [inline Run.command:3]: check if python3 is an executable in $PATH, set flag if it is. I don't think we need the actual path guys 🤔 they could have aliased it. Let the shell resolve it. What do you think?

File: Run.command [hunk 3]
```diff
@@ -1,27 +1,25 @@
...
 python_install=""
 script_name="Scripts/updater.py"
 arg="$1"
 
-if [[ "$py3" == "" ]]; then
-	# No python 3
-	if [[ "$py" == "" ]]; then
-		# No python 2 either
-		echo "Python is not installed!"
+if [[ ! "$python3" ]]; then
```
Comment (rthomazel) [inline Run.command:10]: equivalent to `[[ "$foo" == "" ]]`. Better var names so we don't need the comments

File: Run.command [hunk 4]
```diff
@@ -1,27 +1,25 @@
...
+	if [[ ! "$python" ]]; then
+		echo "Python not found! Please install Python2 or Python3."
```
Comment (rthomazel) [inline Run.command:12]: More helpful message

**Comment by corpnewt** [comment]
Hey @Thomazella - thanks for your contributions to the Run.command!

**Comment by rthomazel** [comment]
Nope. :)

**Pull request merged**